sklearn/mixture/gaussian_mixture.py

```diff
@@ -726,8 +726,7 @@ def bic(self, X):
 
         Returns
         -------
-        bic: float
-            The greater the better.
+        bic: float (the lower the better)
```

Comment:
@t-lanigan  @agramfort is trying to say that you need to follow the indentation. Here you have used the wrong indentation. Please change it as per the review. You'll be good to go.